generated/json_data/qol.jsonc

```diff
@@ -6806,6 +6841,10 @@
                     ]
                 },
                 "Phendrana Shorelines": {
+                    "deleteIds": [
+                        131559, // [Trigger] Trigger - Play Music 1
```

Comment:
If you are fixing music stuff in phen, that okay, just update the PR description to include these changes as well

Reply:
Alternatively, title your commits with specific changes like "remove useless triggers in shorelines" instead of "fix the fixed fixes"

Reply:
I put what I changed in the commit descriptions.